FHEM/lib/SD_ProtocolData.pm

```diff
@@ -1208,9 +1208,9 @@ package lib::SD_ProtocolData;
 				knownFreqs      => '433.92',
 				one             => [7,-1],
 				zero            => [1,-7],
-				start           => [1,-55],
+				start           => [-55],
 				clockabs        => 290,
-#				reconstructBit  => '1',    TODO: Festlegen ob reconstruct ben�tigt wird und wie mit den Startsequenzen zuk�nftig gearbeitet werden soll
+				reconstructBit  => '1',    			# TODO: Festlegen ob reconstruct ben�tigt wird und wie mit den Startsequenzen zuk�nftig gearbeitet werden soll
 				format          => 'tristate', # not used now
```

Comment:
Wie kommt Ihr auf Format tristate?
one und zero sind nur zwei state.
one, zero und float wäre tristate

Reply:
Um ein einheitliches Format zur Übergabe an das Modul beizubehalten, wird erst im Modul in Tristate umgewandelt. Definiert werden die Geräte dann z.B. so: 
Tedsen_SKX6xx_1F10110

Reply:
tristate kann dann ja an dieser Stelle entfallen, wenn es im SIGNALduino Modul nicht umgewandelt wird, was meines Erachtens auch nicht zielführend wäre.